cloudinary-core/src/main/java/com/cloudinary/Uploader.java

```diff
@@ -516,4 +516,23 @@ public String imageUploadTag(String field, Map options, Map<String, Object> html
     public Map deleteByToken(String token) throws Exception {
         return callApi("delete_by_token", ObjectUtils.asMap("token", token), ObjectUtils.emptyMap(), null);
     }
+
+    /**
+     * Populates metadata fields with the given values. Existing values will be overwritten.
+     * @param metadata a map of field name and value.
+     * @param publicIds the public IDs of the resources to update
+     * @param options additional options passed to the request
```

Comment:
What kind of options can I send here? where do I find the list?

Reply:
It's the same options available to all other API (in all SDKs), not sure where it's documented but it's a lot of different keys and not really part of this PR. We can open a separate issue to document all the different possible keys for params and options across the SDK(s).